compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs

```diff
@@ -326,17 +326,24 @@ testCase version getService outdir registerTODO (name, file, anns) = singleTest
 runJqQuery :: (String -> IO ()) -> FilePath -> FilePath -> [String] -> IO [Maybe String]
 runJqQuery log outdir file qs = do
   let proj = takeBaseName file
-  forM qs $ \q -> do
-    log $ "running jq query: " ++ q
-    let jqKey = "external" </> "jq_dev_env" </> "bin" </> if isWindows then "jq.exe" else "jq"
-    jq <- locateRunfiles $ mainWorkspace </> jqKey
-    queryLfDir <- locateRunfiles $ mainWorkspace </> "compiler/damlc/tests/src"
-    let fullQuery = "import \"./query-lf\" as lf; . as $pkg | " ++ q
-    out <- readProcess jq ["-L", queryLfDir, fullQuery, outdir </> proj <.> "json"] ""
-    case trim out of
-      "true" -> pure Nothing
-      other -> pure $ Just $ "jq query failed: got " ++ other
-
+      jsonPath = outdir </> proj <.> "json"
```

Comment:
There's an opportunity for cleanup here since this value is also used as the save location in `jsonSave`, under `mainProj`, in this file. We could instead define it inside `testCase` and pass it to `mainProj` and `runJqQuery`, which would make the connection easier to see.

https://github.com/digital-asset/daml/blob/e82a2042dd405f42448510ad73d741b3474ed3d5/compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs#L470

Reply:
Ah right right, I agree there, it wasn't obvious what was making that file